### PR TITLE
WDPD-207 Improve Credit Card UX

### PR DESCRIPTION
**Pull request by ps-amaureder**
### This PR
* Adds a TODO for Alipay 
* Adds the new string for cc config errors
* Improves cc iframe height for mobile & desktop

### Tests
* The bottom space of the credit card iframe should be the same for mobile and desktop
* Check the error messages for credit card and other payment methods after trying to save a invalid configuration, for credit cards the new string should appear, for all other payment methods nothing should change

### CODE REVIEW

File: Model/PaymentMethod/AlipayCrossBorderPaymentMethod.php [hunk 1]
```diff
@@ -74,6 +74,7 @@ public function getTransaction()
      */
     public function addMandatoryTransactionData(&$oTransaction, $oOrder)
     {
+        //TODO: once account holder is not needed by the sdk, remove this line
```
Comment (ps-amehic) [inline Model/PaymentMethod/AlipayCrossBorderPaymentMethod.php]: What is meant by this?

Comment (ps-dfritzsch) [reply Model/PaymentMethod/AlipayCrossBorderPaymentMethod.php]: The TODO is a reminder to remove the account holder in the future once it is not needed anymore. This was discussed yesterday.

File: views/admin/blocks/wd_admin_payment_main_form.tpl [hunk 2]
```diff
@@ -135,7 +135,11 @@
     <tr>
       <td colspan="2">
         <div class="wdoxidee-messagebox wdoxidee-messagebox--error">
-          [{oxmultilang ident="wd_error_save_failed"}]
+          [{if $paymentMethod->getName() === 'wdcreditcard'}]
```
Comment (tomazpu) [inline views/admin/blocks/wd_admin_payment_main_form.tpl]: So on every save error if the payment method is credit card we show the mismatch url error? Is that correct?

Comment (ps-amaureder) [reply views/admin/blocks/wd_admin_payment_main_form.tpl]: yes

Comment (tomazpu) [reply views/admin/blocks/wd_admin_payment_main_form.tpl]: Can we optimize and check on some code or so? Else DB errors will result in an mismatch error.

Comment (ps-dfritzsch) [reply views/admin/blocks/wd_admin_payment_main_form.tpl]: Good catch, we will have a look.

**Pull request merged**